lib/datadog/appsec/contrib/rack/integration.rb

```diff
@@ -20,7 +22,7 @@ class Integration
           register_as :rack, auto_patch: false
 
           def self.version
-            Gem.loaded_specs['rack'] && Gem.loaded_specs['rack'].version
+            Gem.loaded_specs[Ext::APP] && Gem.loaded_specs[Ext::APP].version
```

Comment:
I experience cognitive dissonance here: the values may be the same (currently) but the semantics is different: this is not the app name but the actual gem name, so the factorisation is incorrect here.